src/main/java/edu/stanford/protege/webprotege/project/IcdReleasedEntityStatusManagerImpl.java

```diff
@@ -0,0 +1,53 @@
+package edu.stanford.protege.webprotege.project;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+
+
+public class IcdReleasedEntityStatusManagerImpl implements IcdReleasedEntityStatusManager {
+
+    @Value("${webprotege.releasedClassesUrl}")
+    private String ICD_RELEASED_URL;
+
+    private final OkHttpClient httpClient = new OkHttpClient();
+
+    private final Logger logger = LoggerFactory.getLogger(IcdReleasedEntityStatusManagerImpl.class);
+
+
+    public IcdReleasedEntityStatusManagerImpl(){}
+
+    @Override
+    public List<String> fetchIris() {
```

Comment:
[Question] will these ever change ? if yes we need to think about a way to do cache bust

Reply:
yes, they will change from time to time. at most 4 times a year from what i understood.